bin/export.py

```diff
@@ -242,6 +242,13 @@ def add_file(path, source):
                     f.unlink()
                     f.write_text(text)
 
+    # TODO: Remove this if we know others use the output_validator dir
+    if (export_dir / "output_validator").exists():
+        (export_dir / "output_validators").mkdir(parents=True)
+        (export_dir / "output_validator").rename(
+            export_dir / "output_validators" / "output_validator"
+        )
```

Comment:
The same problem with symlinks shows up during the export again, but the other way around. The advantage here is that we can try to (recursively) copy the contents of the `output_validator` directory, and hopefully, that will copy the actual file instead of the symlink?

Reply:
oh... did symlinking ever work here since I changed the export code? i could imagine that the symlinks in the tmp dir are already broken?

Reply:
I would hape that it here works and also that the rename is ok since its an absolute symlink from `tmp`->`output_validators` and from there on the relative link should be valid, but i am not sure

Reply:
yes this seems to work here.